README.md

```diff
@@ -37,7 +37,18 @@ This is a work-in-progress method of running a development environment. The stan
 
 One time setup:
 ```bash
+touch ~/.netrc #prevents docker compose from creating it as a directory if you don't have it yet
+
 docker-compose run web bin/rake db:setup
+
+# Run this to get a copy of the Prod database as test data.
+curl -o ./tmp/shared/latest.dump `heroku pg:backups:url -a commitchange`
+
+docker-compose up
+
+# Enter `password` if prompted for a password.
+# This may take up to an hour on Mac.
+docker-compose exec db pg_restore --verbose --clean --no-acl --no-owner -h localhost -U admin -d commitchange_development_legacy /tmp/shared/latest.dump
```

Comment:
Could you explain why this is being done? It sort of recreates `script/restore_from_heroku.sh`.

Reply:
It's because of the `/tmp/shared/latest.dump` path, which differs from the desired path locally. Although, I bet I could make the restore_from_heroku script environment aware... let me play with it.

Reply:
Ok, made a change to use an env var with a  default. Also added different instructions for linux and mac.